Prefabs/Pointers/SharedResources/Scripts/PointerInternalSetup.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace VRTK.Core.Prefabs.Pointers
+{
+    using UnityEngine;
+    using VRTK.Core.Tracking.Follow;
+
+    /// <summary>
+    /// Provides internal settings for the pointer prefabs.
+    /// </summary>
+    public class PointerInternalSetup : MonoBehaviour
+    {
+        /// <summary>
+        /// The <see cref="PointerUserSetup"/> for providing required follow target information.
+        /// </summary>
+        [Tooltip("The Pointer User Setup for providing required follow target information.")]
+        public PointerUserSetup userSetup;
+        /// <summary>
+        /// The <see cref="ObjectFollow"/> component to set the target <see cref="Transform"/> on.
+        /// </summary>
+        [Tooltip("The Object Follow component to set the target Transform on.")]
+        public ObjectFollow objectFollow;
+
+        protected virtual void OnEnable()
+        {
+            if (objectFollow != null)
+            {
+                if (objectFollow.targetComponents.Count == 0)
+                {
+                    objectFollow.targetComponents.Add(userSetup.followTarget);
+                }
+                else
+                {
+                    objectFollow.targetComponents[0] = userSetup.followTarget;

```

Comment:
This is **replacing** the first existing `targetComponent` - is that intended?

Reply:
Yes, it's intended. But I think a better solution is to just clear the list and do an add. There can only be one follow target